docs/topics/kubernetes-developers.md

```diff
@@ -43,6 +43,16 @@ For VERSION, we recommend that you provide a value which would help you identify
 }
 ```
 
+* If the container registry is private, for example Azure Container Registry, then provide the name of the private registry along with the custom hyperkube image like this:
+
+```
+"kubernetesConfig": {
+    "customHyperkubeImage": "<your-private-registry>/hyperkube-amd64:<your-custom-version>",
+    "privateRegistryServer": "<your-private-registry>"

```

Comment:
Does privateRegistryServer make any sense in a context when customHyperkubeImage is not provided? If not, we should probably add validation.